grading/oz/Dockerfile

```diff
@@ -5,24 +5,24 @@ ARG   VERSION=latest
 FROM  ingi/inginious-c-base:${VERSION}
 LABEL org.inginious.grading.name="oz"
 
-RUN     yum -y install \
+RUN     dnf -y install dnf-plugins-core && \
+            dnf config-manager --set-enabled powertools && \
+            dnf -y module enable javapackages-tools && \
+            dnf -y install \
             # Extra dependencies for tools...
             # ... for correctoz (java)
-            java-1.7.0-openjdk java-1.7.0-openjdk-devel \
-            ant ant-junit bc enca wget \
+            java-1.8.0-openjdk java-1.8.0-openjdk-devel \
+            ant-junit bc enca wget \
             # ... for image capture and processing
             ImageMagick Xvfb \
             # ... for sound processing
-            sox lame
+            sox lame-libs
 
-ADD     mozart2-2.0.1+build.8.db41d83-x86_64-centos.rpm /mozart.rpm
-
-RUN     yum -y --nogpgcheck localinstall /mozart.rpm && \
-        rm /mozart.rpm && \
-        yum clean all && \
+RUN     dnf -y --nogpgcheck install https://github.com/mozart/mozart2/releases/download/v2.0.1/mozart2-2.0.1-x86_64-linux.rpm && \
+        dnf clean all && \
         rm -rf /var/cache/yum
```

Comment:
Is it still named /var/cache/yum with dnf?

Reply:
(and is this really needed after the clean all?)

Reply:
I reworked the whole dockerfile as env and aliases were not available as is in ssh mode